pkg/trace/api/api.go

```diff
@@ -516,13 +516,26 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		atomic.AddInt64(&ts.PayloadRefused, 1)
 		return
 	}
+	if err != nil {
+		e := err.Error()
+		if strings.Contains(e, "not found") || strings.Contains(e, "not set") {
+			log.Debugf("Failed to count traces: %s", err)
+		}
+		if strings.Contains(e, "not be parsed") {
```

Comment:
If it value is not set, but the header is there, then that is also an error. The only time we use "debug" is when the header is not set.

Reply:
https://github.com/DataDog/datadog-agent/pull/11804/commits/cf7e155010f85159108a33b322941e2fadaeb994